src/ResourceManagement/Sql/NuGet.Config

```diff
@@ -1,6 +1,6 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<configuration>
-    <config>
-        <add key="repositorypath" value="..\..\..\packages" />
-    </config>
-</configuration>
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Visual Studio must have updated this file and I included the change on accident. I reverted the line endings back to UNIX with commit 0f11fccbbe3b8d14d330aaa299c2820cbf7dc771.